tools/shared.py

```diff
@@ -230,7 +230,7 @@ def run_js_tool(filename, jsargs=[], *args, **kw):
   This is used by emcc to run parts of the build process that are written
   implemented in javascript.
   """
-  command = config.NODE_JS + [filename] + jsargs
+  command = config.NODE_JS + ['--experimental-wasm-bigint'] + [filename] + jsargs
```

Comment:
Is this needed for something?

Reply:
Yes, the JS tools themselves may now contain code that uses bigints?

Reply:
Is that something you have added in this PR?    It seems odd that that would be needed just to process the library js code.. can you point to an example?

Reply:
I'd have to run 64-bit test for that again to uncover that. Can we leave it in? If it's just for the tools it doesn't hurt. Or if you insist I can put it in a comment so it is easier to re-apply when we return to fixing 64-bit issues.